concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -186,14 +190,16 @@ def main():
         force_subtask_results(
             timestamp = timestamp_to_isoformat(current_time),  # current_time
             ack_report_computed_task = ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - 14401),  # current_time - 4:00:01
+                timestamp = timestamp_to_isoformat(current_time - (cluster_consts.subtask_verification_time * 1.4)),
                 subtask_id = task_id,
```

Comment:
If you are adding below `task_id + '2B'` you should do it here as well. Messages should have same ID

Reply:
So it would be best to generate `task_id` for each scenario at the very beginning of it.

Reply:
Done.